addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2545,16 +2545,16 @@ we-button:hover .o_we_shape_animated_label > span {
 // GRID LAYOUT
 we-button.o_grid {
     min-width: fit-content;
-    padding-left: 3px !important;
-    padding-right: 3px !important;
+    padding-left: 4px !important;
+    padding-right: 5px !important;
 }
 
 we-select.o_grid we-toggler {
     width: fit-content !important;
 }
 
-we-button-group.o_grid {
-    min-width: fit-content !important;
+we-button-group.o_grid we-selection-items{
+    width: fit-content !important;
```

Comment:
I think just removing the rule here should solve the issue No? since [".we-selection-items" has already a max-with](https://github.com/odoo/odoo/blob/16.0/addons/web_editor/static/src/scss/wysiwyg_snippets.scss#L1126). Could you confirm this?

Reply:
This rule was needed (in the begining) in order to align the buttons with the inputs below, in order to look better :)  So yes, removing the rule solves the scrollbar issue but then the buttons are not aligned (see image) :grimacing:  
Here, it solves the issue while keeping the visual :wink: 
![image](https://user-images.githubusercontent.com/102796037/233057415-c38a4aa6-6f90-4ff6-8b4c-3f7b72f9a5e5.png)

Reply:
In fact, I notice that it is still slightly not aligned but it is because of the padding value I set above :grimacing: 
I will quickly change the 4px to 4.5px

Reply:
@sobo-odoo Thanks for the explanation. I think `4.5px` on both is better (taking into account that inputs width = `60px*2 + 3px = 123px`) :+1: 
![122](https://user-images.githubusercontent.com/25032876/233356484-864b9446-0438-47c7-a4c6-9ce69c5d7979.png)